ships/shipList.lua

```diff
@@ -0,0 +1,152 @@
+--[[
+    This is a table of all ships available
+    All ships have a:
+    displayName - The display name of the ship.
+    filename - The filename of the ship sprite
+    shape with: 
+        points
+        type
+    spritesheet with:
+        image
+        offset_x
+        scale_x
+    
+        - J.R.C 2/1/22 
+]]
+
+return {
+    { 
+        displayName = 'Trevor\'s OG Ship', -- Index 1
```

Comment:
I noticed you want to keep track of the specific index of each ship. In Lua, there is actually some table syntax you can take advantage of if you want the index displayed in the code:
`[1] = {`, `[2] = {`, etc. all for all of the entries.

Reply:
Oh thank you! I'll make those changes